### PR TITLE
TRAD-141 Add swap stream

### PR DESCRIPTION
**Pull request by yobloxroute**

### CODE REVIEW

**Review by atulsriv — APPROVED** [review]

File: proto/api.proto [hunk 1]
```diff
@@ -974,6 +979,17 @@ message GetQuotesStreamRequest {
   repeated TokenPair tokenPairs = 2;
 }
 
+message GetSwapsStreamRequest {
+  repeated string markets = 1;
```
Comment (aspin) [inline proto/api.proto]: should probably add `repeated Project projects` here as well –– for now only Raydium should be supported, but would be good to get ahead of defining this endpoint

File: proto/api.proto [hunk 2]
```diff
@@ -974,6 +979,17 @@ message GetQuotesStreamRequest {
...
+}
+
+message GetSwapsStreamResponse {
+  int64 slot = 1;
```
Comment (aspin) [inline proto/api.proto]: `Project project` in response too

File: proto/api.proto (hunk 1 above)
Comment (yobloxroute) [reply proto/api.proto]: let's add, thought about that

File: proto/api.proto (hunk 2 above)
Comment (yobloxroute) [reply proto/api.proto]: +

**Review by aspin — APPROVED** [review]

**Pull request merged**